x-pack/plugins/search_assistant/server/routes/index.ts

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
Let's remove this file for now, we don't know whether we want any routes and registering this will get to production, which we also don't want now.

Reply:
@sphilipse removing this file causing [failure in CI ](https://buildkite.com/elastic/kibana-pull-request/builds/228615#01917072-2890-405c-9fbc-865e7de39074)

Reply:
updated in 13b9cb1240a69950192c5ce651be3782b7719b86